library/L1_Peripheral/lpc40xx/test/capture_test.cpp

```diff
@@ -0,0 +1,114 @@
+#include <cstdint>
+#include "L1_Peripheral/cortex/interrupt.hpp"
+#include "L1_Peripheral/interrupt.hpp"
+#include "L0_Platform/lpc40xx/LPC40xx.h"
+#include "L1_Peripheral/lpc40xx/timer.hpp"
+#include "L1_Peripheral/lpc40xx/capture.hpp"
+#include "L4_Testing/testing_frameworks.hpp"
+#include "utility/enum.hpp"
+#include "utility/status.hpp"
+#include "utility/units.hpp"
+
+
+namespace sjsu::lpc40xx
+{
+EMIT_ALL_METHODS(Capture);
+
+TEST_CASE("Testing lpc40xx Capture", "[lpc40xx-capture]")
+{
+    LPC_TIM_TypeDef test_timer_register;
+
+    constexpr int kChannelListSize = 2;
+    constexpr Capture::CaptureChannelNumber kChannelList[kChannelListSize] = {
```

Comment:
```suggestion
    constexpr Capture::CaptureChannelNumber kChannelList[] = {
```
The size constexpr isn't necessary. Remove it.